Lib/asyncio/timeouts.py

```diff
@@ -25,6 +25,7 @@ class _State(enum.Enum):
 
 @final
 class Timeout:
+    """Asynchronous context manager which limits the time spent inside of it."""
 
     def __init__(self, when: Optional[float]) -> None:
```

Comment:
Since most users will just construct the class and not use any of the advanced methods, I recommend having a brief docstring here too. Either this or the class docstring can explain the concept of "`None` means no deadline" so you don't have to repeat that elsewhere.

Reply:
Good idea. I've tried to split that explanation between the initializer and class docstrings, let me know what you think.